### DISCUSSION TITLE
How can I realize that the value of process.env.NODE_ENV can only be production after the project is built? Can it be changed based on what is in the build? For example, let the value be test or other values

### DISCUSSION THREAD

**Discussion by tanglongyan** [nuxt/nuxt#27796] [Questions]
How can I realize that the value of process.env.NODE_ENV can only be production after the project is built? Can it be changed based on what is in the build? For example, let the value be test or other values

**Comment by TheAlexLichter** [comment]
Yes and no. Setting it to production is important to trigger optimizations of various libraries.

But what you want might be https://github.com/nuxt/cli/issues/357#issuecomment-2144498970
